### PR TITLE
docs(ISSUE_TEMPLATE): tweak issue templates

### PR DESCRIPTION
**Pull request by marcoscaceres**
Updates the templates a bit... because, um, 2022?

### CODE REVIEW

**Comment by sidvishnoi** [comment]
Maybe we can use the form style (.yaml) templates now?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -7,16 +7,23 @@ assignees: ""
...
 **Can you or your organization fund the work or help with development?**
-ReSpec is a community maintained project but we don't work for free. To add a new feature, funding pays for the feature development, testing, documentation, server costs, etc. We have an open collective where development costs are tracked and shared. Please see our [Open Collective](https://opencollective.com/respec/).
+ReSpec is a community maintained project, meaning it doesn't have any dedicated staff or funding.
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+There are two options to get your feature implemented. You can:
+
+1. contribute to the project yourself via a pull request. We will [guide](https://respec.org/docs/#developers-guide) you!
+2. fund a contributor to implement your feature (US$500+ depending on complexity).
```
Comment (sidvishnoi) [inline .github/ISSUE_TEMPLATE/feature_request.md:27]: And "just wait"? Maybe instead of "get your featured implemented", say prioritized?

**Comment by sidvishnoi** [comment]
I have tried the .yaml templates in my fork.
Code: https://github.com/sidvishnoi/respec/blob/develop/.github/ISSUE_TEMPLATE/bug_report.yml
Preview: https://github.com/sidvishnoi/respec/issues/new?template=bug_report.yml

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -7,16 +7,23 @@ assignees: ""
...
+
```
Comment (marcoscaceres) [inline .github/ISSUE_TEMPLATE/feature_request.md:28]:
```suggestion
2. fund a contributor to prioritize your feature (usually US$500+ depending on complexity).
3. wait for someone to get to it (usually a few months).
```

**Comment by marcoscaceres** [comment]
oh yeah, neat! are those out of beta yet?

**Comment by sidvishnoi** [comment]
They're in public beta, but looks stable enough. We can keep .md files too. .yml takes preference I think.

**Comment by marcoscaceres** [comment]
Ok, how do you want to proceed? Should we merge this, then update the .yml to match? Or do you want to bring your stuff over to this one?

**Comment by sidvishnoi** [comment]
Yeah lets merge this first. Then we can revert the other yml commit if it causes problems.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -7,16 +7,23 @@ assignees: ""
...
+**Describe any alternatives you've considered**
+How are you currently working around this limitation?
 
 **Can you or your organization fund the work or help with development?**
-ReSpec is a community maintained project but we don't work for free. To add a new feature, funding pays for the feature development, testing, documentation, server costs, etc. We have an open collective where development costs are tracked and shared. Please see our [Open Collective](https://opencollective.com/respec/).
+ReSpec is a community maintained project, meaning it doesn't have any dedicated staff or funding.
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+There are two options to get your feature implemented. You can:
```
Comment (sidvishnoi) [inline .github/ISSUE_TEMPLATE/feature_request.md:24]:
```suggestion
There are three options to get your feature implemented. You can:
```

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**